bulk-model-sync-gradle-test/src/test/kotlin/org/modelix/model/sync/bulk/gradle/test/PushTest.kt

```diff
@@ -40,32 +41,33 @@ class PushTest {
         val repoId = RepositoryId("ci-test")
         val branchName = "master"
         val url = "http://0.0.0.0:${Main.DEFAULT_PORT}/v2"
+
         val branchRef = ModelFacade.createBranchReference(repoId, branchName)
-        val client = ModelClientV2PlatformSpecificBuilder().url(url).build()
+        val client = ModelClientV2PlatformSpecificBuilder().url(url).build().apply { runBlocking { init() } }
+        val replicatedModel = client.getReplicatedModel(branchRef)
+        val branch = runBlocking { replicatedModel.start() }
```

Comment:
Why are these changes necessary? Why are multiple `runBlocking` blocks required instead of only one? Maybe a comment here or in the commit would help understanding this change.

Reply:
The split is necessary to be able to dispose the replicated model. I separated the commits to make it clearer.